src/app/membership/membership.html

```diff
@@ -0,0 +1,88 @@
+                            <a class="btn upgrade-btn" href="#" data-navigo>Get started</a>
+                        </div>
+                    </div>
+                </div>
+                <div class="col-lg-4 box">
+                    <div class="bg-white mt-3 price-box">
+                        <div class="pricing-name text-center">
+                            <h4 class="mb-0 plan-title">Platinum</h4>
+                        </div>
+                        <div class="plan-price text-center mt-4">
+                            <h1 class="text-custom font-weight-normal mb-0"><b>€690</b><span>/Year</span></h1>
+                        </div>
+                        <div class="price-features mt-5">
+                            <p><i class="mdi mdi-check"></i> 3 Products <span class="font-weight-bold">FOR FREE</span></p>
+                            <p><i class="mdi mdi-check"></i> 5 Extra services <span class="font-weight-bold">FOR FREE</span></p>
+                            <p><i class="mdi mdi-check"></i> Other services <span class="font-weight-bold">25% OFF</span></p>
+                            <p><i class="mdi mdi-check"></i> All services <span class="font-weight-bold">24/YEAR</span></p>
+                            <p><i class="mdi mdi-check"></i> Visit every <span class="font-weight-bold">2-4 WEEKS</span></p>
+                        </div>
+                        <div class="text-center mt-5">
+                            <a class="btn upgrade-btn" href="#" data-navigo>Get started</a>
+                        </div>
+                    </div>
+                </div>
+                <div class="col-lg-4 box">
+                    <div class="bg-white mt-3 price-box">
+                        <div class="pricing-name text-center">
+                            <h4 class="mb-0 plan-title">Gold</h4>
+                        </div>
+                        <div class="plan-price text-center mt-4">
+                            <h1 class="text-custom font-weight-normal mb-0"><b>€450</b><span>/Year</span></h1>
+                        </div>
+                        <div class="price-features mt-5">
+                            <p><i class="mdi mdi-check"></i> 2 Products <span class="font-weight-bold">FOR FREE</span></p>
+                            <p><i class="mdi mdi-check"></i> 4 Extra services <span class="font-weight-bold">FOR FREE</span></p>
+                            <p><i class="mdi mdi-check"></i> Other services <span class="font-weight-bold">20% OFF</span></p>
+                            <p><i class="mdi mdi-check"></i> All services <span class="font-weight-bold">16/YEAR</span></p>
+                            <p><i class="mdi mdi-check"></i> Visit every <span class="font-weight-bold">3-5 WEEKS</span></p>
+                        </div>
+                        <div class="text-center mt-5">
+                            <a class="btn upgrade-btn" href="#" data-navigo>Get started</a>
```

Comment:
Proper link example:

`<a class="btn upgrade-btn" href="/dashboard/membership/gold" data-navigo>Get started Gold</a>`

Same principle applies to the other membership packages, just change the last part of `href`:

`<a class="btn upgrade-btn" href="/dashboard/membership/silver" data-navigo>Get started Silver</a>`

Reply:
Done

Reply:
Resolved.